File: coding-test/백준/사칙연산.py
# https://www.acmicpc.net/problem/10869

# 2024.02.14

import sys
import math

n, m = list(map(int, sys.stdin.readline().split()))
# n, m = list(map(int, '7 3'.split()))

print(n+m)
print(n-m)
print(n*m)
print(math.floor(n/m))
print(n%m)

File: coding-test/백준/사칙연산.js
// https://www.acmicpc.net/problem/10869

/**
 * @Date 2024.02.14
 */

const fs = require('fs');
const [a, b] = fs.readFileSync('/dev/stdin').toString().split(' ').map(Number);

console.log(a + b);
console.log(a - b);
console.log(a * b);
console.log(Math.floor(a / b));
console.log(a % b);
